packages/waku/src/lib/plugins/vite-plugin-fs-router-typegen.ts

```diff
@@ -142,9 +142,10 @@ export const fsRouterTypegenPlugin = (opts: { srcDir: string }): Plugin => {
               hasGetConfig,
             });
           } else if (filePath.endsWith('/index.tsx')) {
+            const path = filePath.slice(0, -'/index.tsx'.length);
             fileInfo.push({
               type: 'page',
-              path: filePath.replace('index.tsx', ''),
+              path: path.length || '/',
```

Comment:
.length seems a bug.
```suggestion
              path: path || '/',
```
or `path: path.length ? path : '/'`.

Reply:
whoops, thanks